src/function/numeric/solveODE.js

```diff
@@ -0,0 +1,304 @@
+import { isUnit, isBigNumber } from '../../utils/is.js'
+import { factory } from '../../utils/factory.js'
+
+const name = 'solveODE'
+const dependencies = [
+  'typed',
+  'add',
+  'subtract',
+  'multiply',
+  'divide',
+  'max',
+  'map',
+  'abs',
+  'isPositive',
+  'isNegative',
+  'isNumeric',
+  'larger',
+  'smaller',
+  'matrix',
+  'bignumber'
+]
+
+export const createSolveODE = /* #__PURE__ */ factory(name, dependencies, (
+  {
+    typed,
+    add,
+    subtract,
+    multiply,
+    divide,
+    max,
+    map,
+    abs,
+    isPositive,
+    isNegative,
+    isNumeric,
+    larger,
+    smaller,
+    matrix,
+    bignumber
+  }
+) => {
+  /**
+     * Numerical Integration of Ordinary Differential Equations
+     *
+     * Two variable step methods are provided:
+     * - "RK23": Bogacki–Shampine method
+     * - "RK45": Dormand-Prince method RK5(4)7M (default)
+     *
+     * The arguments are expected as follows.
+     *
+     * - `func` should be the forcing function `f(t, y)`
+     * - `tspan` should be a vector of two numbers or units `[tStart, tEnd]`
+     * - `y0` the initial state values, should be a scalar or a flat array
+     * - `options` should be an object with the following information:
+     *   - `method` ('RK45'): ['RK23', 'RK45']
+     *   - `tol` (1e-3): A numeric value
```

Comment:
Can you explain what `tol` is?  (i.e. the tolerance, and that the function keeps iterating until the result has this minimal tolerance)

Reply:
Yes of course

Reply:
Done